cabal-testsuite/PackageTests/PreProcess/Hsc2HsOptionsCC/setup.test.hs

```diff
@@ -4,25 +4,25 @@ import Data.Maybe (isJust)
 import System.Directory (findExecutable)
 
 -- Check that preprocessors (hsc2hs) are run
-main = setupAndCabalTest $ do
-    -- we need "g++"
-    hasGxx <- liftIO $ fmap isJust $ findExecutable "g++"
-    skipUnless "g++" hasGxx
-
+main = do
     -- Figure out how recent GHC we need
     -- https://github.com/msys2/MINGW-packages/issues/3531
-    skipIfWindows
+    skipIfWindows "Maybe? need to try with g++"
```

Comment:
TODO try this with g++. It is problematic because the CCCompilerOverride test requires clang 🙄

Reply:
g++ is problematic on Windows since ghc moved to the clang-based toolchain.